openstates/la/committees.py

```diff
@@ -47,15 +48,19 @@ def scrape_senate_committee(self, name, url):
         text = self.urlopen(url)
         page = lxml.html.fromstring(text)
 
-        links = page.xpath('//table[@bordercolor="#EBEAEC"]/tr/td/font/a')
+        links = page.xpath(
```

Comment:
I think this xpath might be too broad - it's pulling in the links from the sidebar on this page, which is causing it to fail locally for me:

http://senate.la.gov/Committees/default.asp?type=Standing

Let me see if I can find an xpath that works.